docs/source/user_guide/interoperability.rst

```diff
@@ -155,20 +155,16 @@ PyTorch also supports zero-copy data exchange through ``DLPack`` (see :ref:`dlpa
 	import cupy
 	import torch
 
-	from torch.utils.dlpack import to_dlpack
-	from torch.utils.dlpack import from_dlpack
+	from torch import from_dlpack
```

Comment:
My apology, would you mind making it even simpler? I should have thought about this sooner 🙂 
```suggestion
```